dev/com.ibm.ws.concurrent_fat/test-bundles/test.resource/src/test/resource/internal/TestResource.java

```diff
@@ -68,12 +66,18 @@ public Object run() throws Exception {
                     if (!"com.ibm.ws.threading.internal.ScheduledExecutorImpl".equals(componentName))
                         throw new Exception("Unexpected ScheduledExecutorService with highest service.ranking: " + schedExecSvcRef);
 
-                    ServiceReference<ManagedExecutorService> mgdExecSvcRef = bundleContext.getServiceReference(ManagedExecutorService.class);
+                    ServiceReference<?> mgdExecSvcRef = bundleContext.getServiceReference("jakarta.enterprise.concurrent.ManagedExecutorService");
+                    if (mgdExecSvcRef == null)
+                        mgdExecSvcRef = bundleContext.getServiceReference("javax.enterprise.concurrent.ManagedExecutorService");
+
                     String displayId = (String) mgdExecSvcRef.getProperty("config.displayId");
                     if (!"managedExecutorService[DefaultManagedExecutorService]".equals(displayId))
                         throw new Exception("Unexpected ManagedExecutorService with highest service.ranking: " + displayId);
 
-                    ServiceReference<ManagedScheduledExecutorService> mgdSchedExecSvcRef = bundleContext.getServiceReference(ManagedScheduledExecutorService.class);
+                    ServiceReference<?> mgdSchedExecSvcRef = bundleContext.getServiceReference("java.enterprise.concurrent.ManagedScheduledExecutorService");
```

Comment:
Missing "x" in "javax".  Should be:
javax.enterprise.concurrent.ManagedScheduledExecutorService

Reply:
Good catch - added a commit to fix it.